docs/management_api.yml

```diff
@@ -38,6 +38,10 @@ responses:
     description: Invalid Request.
     schema:
       $ref: "#/definitions/Error"
+  ConflictError: # 409
+    description: Conflict
```

Comment:
What about being more specific here?
Like `An active deployment with the exact same parameters already exists`?

Reply:
Updated